cmd/security-secretstore-setup/entrypoint.sh

```diff
@@ -47,6 +47,11 @@ echo "Initializing secret store"
 echo "Executing custom command: $@"
 "$@"
 
+# Share Redis v5 password
+mkdir -p /secrets
+chown -Rh 100:1000 /secrets
+/security-secretstore-read -confdir /res-read >${REDIS5_PASSWORD_PATHNAME}
+
```

Comment:
This is buggy.  Presuming from the above, that the secret store is the Record of Origin for the redis password, we shouldn't be taking it out of the secret store and putting it in a persistent docker volume.  This means that `REDIS5_PASSWORD_PATHNAME` should probably be of the form /tmp/edgex/secrets/edgex-redis/redis5-password  instead of /secrets/redis5-password.  Moreover, we should be guarding this with an "if" statement and not execute this block of code if `REDIS5_PASSWORD_PATHNAME` is not defined.  Also we should not assume the directory is /secrets but should be computing `basename ${REDIS5_PASSWORD_PATHNAME}` and chmodding from there.

Reply:
WIP

Reply:
Done